File: ldfs/src/main/kotlin/com/ldfs/control/domain/service/leaderElection/LeaderElectionServiceConfig.java
package com.ldfs.control.domain.service.leaderElection;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.client.RestTemplate;

@Configuration
public class LeaderElectionServiceConfig {
    @Bean
    public RestTemplate restTemplate() {
        return new RestTemplate();
    }
}

File: ldfs/src/main/kotlin/com/ldfs/control/domain/service/leaderElection/LeaderElectionServiceConfig.kt
package com.ldfs.control.domain.service.leaderElection

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.web.client.RestTemplate

@Configuration
class LeaderElectionServiceConfig {
    @Bean
    fun restTemplate(): RestTemplate {
        return RestTemplate()
    }
}